src/librustdoc/html/render/mod.rs

```diff
@@ -1131,7 +1131,7 @@ fn render_assoc_items_inner(
             write!(
                 w,
                 "<h2 id=\"trait-implementations\" class=\"small-section-header\">\
-                     Trait Implementations<a href=\"#trait-implementations\" class=\"anchor\"></a>\
+                     Traits<a href=\"#trait-implementations\" class=\"anchor\"></a>\
```

Comment:
That's a bit weird when reading the sidebar because it doesn't make difference anymore between "traits implementation" and "traits in the same module" which is below.

Reply:
They're differentiated by position in the hierarchy, though.

Reply:
What about writing "Trait Impls" instead? I think that'd be the most idiomatic.

Reply:
I think "Trait Impls" is idiomatic in the sense that jargon is usually idiomatic. But we should avoid jargon in the docs and prefer to match what's in The Book and The Reference where possible. So my preference order would be:

1. Traits
2. Trait Implementations // I'm totally fine with this one!
3. Trait Impls // less preferred

Reply:
`impl` is the actual keyword; in some ways, it seems clearer to me than "implementations", and it's certainly shorter.

"Traits" seems slightly confusing to me, especially since traits may be listed multiple times in that list (with different generics).

Reply:
It's the keyword, but the keyword is never plural (and is usually seen in monospace). I got no hits on "impls" in The Book, and just one hit in The Reference.

Reply:
Anyhow, I do see your point about "Traits" being confusing. How do you feel about switching this back to "Trait Implementations"?

Reply:
Yeah, I'd like to have more discussion about this at some point (I also feel that "Trait Implementations" is overly verbose), but in the interest of not making too many changes at once, let's leave it as-is.